ebos/eclwriter.hh

```diff
@@ -420,7 +387,10 @@ public:
                                 isFlowsn, std::move(flowsn),
                                 isFloresn, std::move(floresn));
         }
+
```

Comment:
Better to avoid whitespace changes in places not related to the current changes. Instead make such changes in a consistent manner in a separate commit and PR, if they are necessary. Also lines 392-393.